### PR TITLE
tidying

### PR DESCRIPTION
**Pull request by ikona23**
I wrapped everything into id of the body. And continued till everything was ok :). Commenting really helped to know where am I, of course good IDE. (thank you ATOM), when I double clicked on div it will show me where is the end of the tag. Please review my work and let me know what needs to be fixed.

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+            "http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">
+            Zen Garden, the book</a>.</p>
+        </div>
+        <footer>
+            <a class="zen-validate-html" href=
+            "http://validator.w3.org/check/referer" title=
+            "Check the validity of this site&#8217;s HTML">HTML</a>
+            <a class="zen-validate-css" href=
+            "http://jigsaw.w3.org/css-validator/check/referer" title=
+            "Check the validity of this site&#8217;s CSS">CSS</a> <a class=
```
Comment (guanzhou-zhao) [inline pristine-zen-garden.html]: @ikona23 
About #1 , the code looks pretty good. Especially that the lines have been wrapped.

If you can put the last 3 <anchor> tags in <footer> tag to new lines, that'd be perfect and we can simulate the full cycle of "pull request".

Cheers, Ben

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+            "http://creativecommons.org/licenses/by-nc-sa/3.0/" title=
+            "View the Creative Commons license of this site: Attribution-NonCommercial-ShareAlike.">
+            CC</a> <a class="zen-accessibility" href=
+            "http://mezzoblue.com/zengarden/faq/#aaa" title=
+            "Read about the accessibility of this site">A11y</a> <a class=
+            "zen-github" href=
+            "https://github.com/mezzoblue/csszengarden.com" title=
+            "Fork this site on Github">GH</a>
+        </footer>
+    </div>
```
Comment (guanzhou-zhao) [inline pristine-zen-garden.html:181]: @ikona23 
About #1 , the code looks pretty good. Especially that the lines have been wrapped.

If you can put the last 3 &lt;anchor&gt; tags in &lt;footer&gt; tag to new lines, that'd be perfect and we can simulate the full cycle of "pull request".

Cheers, Ben

Comment (ikona23) [inline pristine-zen-garden.html:181]: All done, Ben. Thanks for pointing that out.

Regards Peter

**Comment by harrison-symes** [comment]
Reviewed this code, no issues as far as I can see after Ben's correction. good stuff Peter!

**Pull request merged**